### PR TITLE
Fix/not call with open circuit

### PR DESCRIPTION
**Pull request by duilio92**
I added a test to check the error described, and fix it returning None to finish execution after calling fallback function.

### CODE REVIEW

File: circuitbreaker.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ def call(self, func, *args, **kwargs):
         if self.opened:
             if self.fallback_function:
                 self.fallback_function(*args, **kwargs)
+                return None
```
Comment (luiscoms) [inline circuitbreaker.py:65]: Better return the fallback function return

```python
return self.fallback_function(*args, **kwargs)
```

File: circuitbreaker.py [hunk 2]
```diff
@@ -62,6 +62,7 @@ def call(self, func, *args, **kwargs):
...
             else:
```
Comment (luiscoms) [inline circuitbreaker.py:66]: Else is not necessary

File: circuitbreaker.py (hunk 1 above)
Comment (duilio92) [reply circuitbreaker.py:65]: ok, thanks will fix!

File: circuitbreaker.py (hunk 2 above)
Comment (duilio92) [reply circuitbreaker.py]: Okey, fixed

File: tests/test_unit.py [hunk 3]
```diff
@@ -55,6 +55,17 @@ def test_circuitbreaker_should_call_fallback_function_if_open():
     cb.call(func)
     fallback.assert_called_once_with()
 
+def test_circuitbreaker_should_not_call_function_if_open():
+    fallback = Mock(return_value=True)
```
Comment (luiscoms) [inline tests/test_unit.py:59]: Test return of fallback function

Comment (duilio92) [reply tests/test_unit.py:59]: Thanks for the correction! fixed

File: circuitbreaker.py (hunk 1 above)
Comment (duilio92) [reply circuitbreaker.py]: Fixed

**Comment by duilio92** [comment]
All corrections have been fixed, thanks for the comments!

**Comment by fabfuel** [comment]
Thanks for the fix @duilio92 & thanks for the review @luiscoms

**Pull request merged**